src/views/Nominate.vue

```diff
@@ -1,59 +1,89 @@
 <template>
-    <div class="nominate">
-        <div class="nominate__section">
-            <div class="nominate__content-wrapper">
-                <h1 class="nominate__title">
-                    Nominate a <span class="nominate__highlight">developer</span>
-                </h1>
-                <p class="nominate__body-text">
-                    Help us build the directory of Developers thriving in the field of programming.
-                    If you know someone who deserves to be on the list, nominate them by clicking the Nominate button and filling the form.
-                    And hey, it's alright to nominate yourself. 😎
-                </p>
-                <a href="#" rel="noreferrer" class="btn primary">Nominate</a>
-            </div>
-        </div>
+  <div class="nominate">
+    <div class="nominate__section">
+      <div class="nominate__content-wrapper">
+        <h1 class="nominate__title">
+          Nominate a <span class="nominate__highlight">developer</span>
+        </h1>
+        <p class="nominate__body-text">
+          Help us build the directory of Developers thriving in the field of
+          programming. If you know someone who deserves to be on the list,
+          nominate them by clicking the Nominate button and filling the form.
+          And hey, it's alright to nominate yourself. 😎
+        </p>
+
+        <a href="#nominate" rel="noreferrer" class="btn primary" @click="onNominar">
```

Comment:
Because this is pretty simple, we don't necessarily need to create a method for that. We can just inline it.

```suggestion
        <a href="#nominate" rel="noreferrer" class="btn primary" @click="isOpen != isOpen">
```